sinch/domains/voice/models/applications/responses.py

```diff
@@ -0,0 +1,40 @@
+from typing import List
+
+from dataclasses import dataclass
+from sinch.core.models.base_model import SinchBaseModel
+from sinch.domains.voice.models import ApplicationNumber, Price
+
+
+@dataclass
+class GetNumbersVoiceApplicationResponse(SinchBaseModel):
+    numbers: List[ApplicationNumber]
+
+
+@dataclass
+class AssignNumbersVoiceApplicationResponse(SinchBaseModel):
+    pass
+
+
+@dataclass
+class UnassignNumbersVoiceApplicationResponse(SinchBaseModel):
+    pass
+
+
+@dataclass
+class UpdateCallbackUrlsVoiceApplicationResponse(SinchBaseModel):
+    pass
+
+
+@dataclass
+class GetCallbackUrlsVoiceApplicationResponse(SinchBaseModel):
+    primary: str
+    fallback: str
+
+
+@dataclass
+class QueryNumberVoiceApplicationResponse(SinchBaseModel):
+    country_id: str
+    number_type: str
```

Comment:
Candidate for a literal too

Reply:
This a `response` kind of object (it's created solely by the SDK). Type hints are not needed there. Maybe leaving this property in more "open" manner is safer in terms of API responses handling?